solr/modules/ltr/src/java/org/apache/solr/ltr/model/MultipleAdditiveTreesModel.java

```diff
@@ -361,25 +372,25 @@ private static String explainNode(float[] featureVector, RegressionTreeNode regr
       // each branch and report
       // that here
 
-      if (featureVector[regressionTreeNode.featureIndex] <= regressionTreeNode.threshold) {
-        returnValueBuilder.append(
-            "'"
-                + regressionTreeNode.feature
-                + "':"
-                + featureVector[regressionTreeNode.featureIndex]
-                + " <= "
-                + regressionTreeNode.threshold
-                + ", Go Left | ");
+      if ((featureVector[regressionTreeNode.featureIndex] <= regressionTreeNode.threshold) ||
```

Comment:
restructure here, with enums and potentially leave this block unchanged and add a block for null values, also in the explain, explicitly mention: the feature has null value, go right bla bla"
THis is for explainability, so, explain it as much as possible!